ansible/roles/ocp4-workload-integreatly/defaults/main.yml

```diff
@@ -24,4 +24,27 @@ silent: False
 ocp4_workload_integreatly_defaults:
     variable_1: "value 1"
     variable_2: "value 2"
-    variable_secret: ""
\ No newline at end of file
+    variable_secret: ""
+
+action: install
+namespace: redhat-rhmi-3scale
+
+seed_secret_name: system-seed
+
+tmp_dir: /tmp
+
+# default tenant details
+tenant_name: admin-tenant
+tenant_admin_user: admin
+tenant_admin_email: admin@example.com
+tenant_admin_password: Password1
```

Comment:
We will probably want to generate this password.

Reply:
Think your talking about https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles/ocp-workload-integreatly/tasks/workload.yml#L9L13 but this is just a placeholder so the ansible doesn't fall over